niri-config/src/lib.rs

```diff
@@ -323,8 +323,8 @@ pub struct Output {
     pub position: Option<Position>,
     #[knuffel(child, unwrap(argument, str))]
     pub mode: Option<ConfiguredMode>,
-    #[knuffel(child)]
-    pub variable_refresh_rate: bool,
+    #[knuffel(child, default)]
+    pub variable_refresh_rate: Vrr,
```

Comment:
I think it would be clearer, code- and type-wise, if this worked like FocusFollowsMouse. That is, `pub variable_refresh_rate: Option<Vrr>` where None means disabled, Some means enabled, then on_demand is a bool with default value false.

Reply:
That was my initial implementation but the downside is no help function, which is why I wrapped it around a new type.